Mage.Sets/src/mage/cards/c/CleansingBeam.java

```diff
@@ -41,11 +41,7 @@ public CleansingBeam copy() {
 
 class CleansingBeamEffect extends OneShotEffect {
 
-    static final FilterPermanent filter = new FilterPermanent("creature");
-
-    static {
-        filter.add(CardType.CREATURE.getPredicate());
-    }
+    static final FilterCreaturePermanent filter = new FilterCreaturePermanent();
```

Comment:
(can be simplified even further, just call `StaticFilters.FILTER_PERMANENT_CREATURE` where used)